src/test_utils/oauth_server.ts

```diff
@@ -5,6 +5,6 @@ export const startOauthServer = async () => {
   const server = new OAuth2Server();
   await server.issuer.keys.generate('RS256');
   await server.start(0, 'localhost');
-  log.info('Started OAuth server');
+  log.debug('Started OAuth server');
```

Comment:
Took the liberty to squeeze this in, didn't think every test run should have logged these.